src/components/footer/Footer.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Footer.css';
+
+
+const Footer = () => (
```

Comment:
I would consider injecting things into this component with props instead of hard coding